junction/profiles/views.py

```diff
@@ -22,3 +26,15 @@ def dashboard(request):
                 conf_proposals[conf.name] = [proposal]
     return render(request, 'profiles/dashboard.html',
                   {'conf_proposals': conf_proposals})
+
+
+@login_required
+def profile(request):
+    if request.method == "POST":
+        city = request.POST.get('city')
+        contact_no = request.POST.get('contact_no')
+        Profile.objects.create(user=request.user, city=city, contact_no=contact_no)
+        return HttpResponseRedirect("/profiles")
+
+    elif request.method == "GET":
+        return render(request, 'profiles/userprofile.html')
```

Comment:
If the user has already created the profile, existing values should be filled in the form. So pass the profile object to HTML.